### PR TITLE
refactor: extract contributor

### PR DESCRIPTION
**Pull request by xiaowangxiaowang256256**
Adjust the style of contributor.

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
Please feel free to give us a screenshot of the changes.

File: src/views/contributor.vue [hunk 1]
```diff
@@ -26,40 +26,66 @@ if (contributor.value)
...
-        </a>
-      </span>
+      <!-- title -->
+      <span v-if="contributor.title">{{ contributor.title }}</span>
 
-      <component :is="contributor.default" />
+      <!-- social -->
+      <div>
+        <span v-if="contributor.github">
+          Github:
```
Comment (sxzz) [inline src/views/contributor.vue:49]: Use an icon for social media instead
```suggestion
          GitHub:
```

**Comment by xiaowangxiaowang256256** [comment]
> Please feel free to give us a screenshot of the changes.

Before the  change: information is arranged on the left.
0a8c0e3ab497263c735ebf3b8c6e42a
591404010e34d1213415e930b39c199

Before the  change: information is arranged on the center.
25061e5222af73e033b56c985b66eb0
aaa58435b473d9fcfbf267636e89714

File: src/views/contributor.vue (hunk 1 above)
Comment (xiaowangxiaowang256256) [reply src/views/contributor.vue]: > Use an icon for social media instead

Adjustment completed
image

**Review by wey-gu — APPROVED** [review]
Great change, thanks 🫡

**Comment by LinuxSuRen** [comment]
This PR is great. It looks better. Actually we could have a people list page to show all contributors of this progam which include guests, hosts, and GitHub contributors. People is always the most important part.

**Pull request merged**